docs/root/api-docs/xds_protocol.rst

```diff
@@ -37,6 +37,28 @@ appears below, and takes the form ``type.googleapis.com/<resource type>`` -- e.g
 ``type.googleapis.com/envoy.config.cluster.v3.Cluster`` for a ``Cluster`` resource. In various requests from
 Envoy and responses by the management server, the resource type URL is stated.
 
+Protoc-Gen-Validate Annotations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The protobuf messages for the individual xDS resource types have annotations
+using `protoc-gen-validate<https://github.com/envoyproxy/protoc-gen-validate>`
+(PGV), which indicate semantic constraints to be used to validate the contents
+of a resource when it is received by a client.
+
+Clients are not required to use these PGV annotations to validate the
+resources (e.g., Envoy does this validation, but gRPC does not). Also, the
+PGV annotations are not intended to be an exhaustive list of validation checks
+to be performed by the client; clients may reject a resource for reasons
+unrelated to the PGV annotations.
+
+In general, it is not recommended that control planes or xDS proxies directly
```

Comment:
I'm not sure I'd put it exactly this way. Control planes and xDS proxies shouldn't blindly perform PGV validation at some random version of the API. However, if they have a range of supported clients, they are free to perform PGV validation on this range and use this as a means to sanity check config. In the same way they might actually test out the configuration on the range of clients in a heavier weight config validation.

What is definitely true is that the PGV validations are not intended for consumption by the control plane - the only value is as a method to catch potential rejection from clients.

Reply:
I've updated the wording to attempt to better capture the intent here.  PTAL.